docs/linguistic-data-intro.md

```diff
@@ -133,3 +158,89 @@ by making sure every row has the same number of tabs.
 
 By default it will reformat every `altlab.tsv` file, but you can explicitly
 give it the path to any TSV file on the command line.
+
+# Abstract data
+
+Having the following information available in some form is *extremely
+useful* for developers, community members, and other linguists who are not
+as familiar with the language, or your conventions for describing it, as
+you are.
+
+This may be in the form of prose documents, spreadsheets, diagrams, and so
+on, as long as the intent is to provide reference material for humans to
+understand what is going on.
+
+  - **Part-of-speech categories** and the relations among them
```

Comment:
Would here be a good place for that screenshot of the part-of-speech ontology we took?